controllers/btpoperator_controller.go

```diff
@@ -1073,6 +1127,12 @@ func (r *BtpOperatorReconciler) watchSecretPredicates() predicate.Funcs {
 			if oldSecret.Name == SecretName && oldSecret.Namespace == ChartNamespace {
 				return true
 			}
+			if oldSecret.Name == CaSecret && oldSecret.Namespace == ChartNamespace {
```

Comment:
If you do extraction suggested in previous comment you could just call the same function passing oldSecret as parameter.